ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -208,13 +220,26 @@ const Frequencies = React.memo(({ variant }) => {
           ))}
         </Popup.Content>
         {hasGlobalAfPops.length > 0 && hasAcPops.length > 0 && <VerticalSpacer height={5} />}
-        {hasAcPops.length > 0 && <Popup.Header content="Allele Counts" />}
+        {(hasAcPops.length > 0 || hasAcHetPops.length > 0 || hasMaxHlPops.length > 0) && <Popup.Header content="Allele Counts" />}
         <Popup.Content>
           {hasAcPops.map(pop => (
             <div key={pop.field}>
               {`${pop.fieldTitle}: ${populations[pop.field].ac} out of ${populations[pop.field].an}`}
             </div>
           ))}
+          {hasAcHetPops.map(pop => (
```

Comment:
these aren't allele counts, you need to add a different header and section for these

Reply:
How about adding a header "Heteroplasmic statistics" for displaying ac_het/af_het/max_hl?

Reply:
I think just "Heteroplasmy" should be fine